### PR TITLE
Adding Package xcube_geodb

### PR DESCRIPTION
**Pull request by dzelge**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/xcube-geodb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - pip
+    - xcube >=0.3.0
+    - geopandas >=0.5
+    - matplotlib
```
Comment (scopatz) [inline recipes/xcube-geodb/meta.yaml]: Can this be matplotlib-base or does it need the Qt backend?

File: recipes/xcube-geodb/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  license_family: MIT
+  license_file: LICENSE
+  description: "geodb is a Python package"
+  doc_url: https://xcube_geodb.readthedocs.io
+  dev_url: https://github.com/dcs4cop/xcube-geodb
+
+extra:
+  recipe-maintainers:
+    - dzelge
+    - forman
```
Comment (scopatz) [inline recipes/xcube-geodb/meta.yaml]: @forman are you OK with being a maintainer?

Comment (forman) [reply recipes/xcube-geodb/meta.yaml]: @scopatz sure, thanks!

File: recipes/xcube-geodb/meta.yaml (hunk 1 above)
Comment (forman) [reply recipes/xcube-geodb/meta.yaml]: matplotlib-base should be fine

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**